api/test/integration/env/docker-compose.yml

```diff
@@ -112,16 +104,13 @@ services:
       - nginx-lb
       - wazuh-agent3
     depends_on:
-      - wazuh-agent2
+      - wazuh-agent1
```

Comment:
All agents using the `new` image must depend on the agent which is building it (agent1), the same applies for the old image (agent5)